CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.62.0](https://github.com/rudderlabs/rudder-transformer/compare/v1.60.0...v1.62.0) (2024-04-15)
+
+
```

Comment:
Duplicate change log again - @sanpj2292 can you check any action updates as part of CI/CD project leading to this ?

Reply:
yeah but why that's happening ? It's the same case from last 3 releases - why just the previous version is not being considered ?